### DISCUSSION TITLE
Does Cypress servers store the test execution information if not enrolled to "Cypress Cloud"?

### DISCUSSION THREAD

**Discussion by stoonya** [cypress-io/cypress#25403] [General]
Hi everyone,

My company is strict regarding data sensitivity and security, and we are considering Cypress as a test automation tool. 

I am wondering if Cypress sends/stores any logs, test execution artifacts, or any other information on their servers (basically outside the local machine where the test was executed) if the team is not enrolled in the "Cypress Cloud"? 

The https://www.cypress.io/security/ says that they store the test execution information on the Cypress AWS cloud if the project is enrolled to "Cypress Cloud", but they don't say explicitly that they don't do that for not enrolled projects.

Thanks!

**Comment by emilyrohrbough** [comment]
@stoonya If you are not recording a run to the Cloud, no data is collected by Cypress. Even when signed up to use the Cloud, the run must use the `--record` flag for data to be collected by Cypress.
